client/src/components/ButtonContainer.jsx

```diff
@@ -4,8 +4,32 @@ import styled from 'styled-components';
 const StyledButtonContainer = styled.div`
   display: flex;
   justify-content: flex-start;
+  margin-top: 3.575rem
 `;
 
+const StyledButton = styled.button`
+  background-color: black;
+  min-width: 14rem;
+  border-color: transparent;
+  border-radius: 3rem;
+  font-family: 'Nunito Sans';
+  font-size: 1.25rem;
+  font-weight: bold;
+  color: white;
+  line-height: 3.6rem;
+`;
+
+const TextStyle = styled.span`
+  margin: 0px 0px 0px 32px;
+  border-color: transparent;
+  border-radius: 3rem;
+  font-family: 'Nunito Sans';
+  font-size: 1.25rem;
+  font-weight: bold;
```

Comment:
I think you can use the 700 instead of bold. I noticed you grabbed 400 and 700 for the fonts.